pkgs/halo2-ecdsa/src/circuits/efficient_ecdsa.rs

```diff
@@ -25,9 +25,9 @@ where
     SF: BigPrimeField,
     GA: CurveAffineExt<Base = CF, ScalarExt = SF>,
 {
-    s: SF,
-    T: GA,
-    U: GA,
+    pub s: SF,
+    pub T: GA,
+    pub U: GA,
```

Comment:
@duguorong009 why is this change?

Reply:
This `EthMembershipInputs` is used in `EthMembershipCircuit` of `halo2-eth-membership` pkg/crate.
When used, the fields should be accessed.
https://github.com/anonklub/anonklub/blob/feature/halo2-eth-membership/pkgs/halo2-eth-membership/src/eth_membership.rs#L135
https://github.com/anonklub/anonklub/blob/feature/halo2-eth-membership/pkgs/halo2-eth-membership/src/eth_membership.rs#L177-L186

Reply:
Ah okay I see now. LGTM!